src/_gettsim_tests/test_renten_alter.py

```diff
@@ -42,14 +43,61 @@
 
 @pytest.fixture(scope="module")
 def input_data():
-    file_name = "renten_alter.csv"
-    out = pd.read_csv(TEST_DATA_DIR / file_name)
+    # Original Data Source: Seibold, A. (2021).
+    # Reference points for retirement behavior:
+    # Evidence from german pension discontinuities.
+    # American Economic Review, 111(4), 1126-65.
+
+    # IDs were added to the original data
+    # m_pflichtbeitrag is added as an identifier for pension kind
+    # 60: Regelaltersrente, 420: Langjährig Versichert, 180: Altersrente für Frauen
+    original_data = pd.read_csv(TEST_DATA_DIR / "renten_alter.csv")
+
+    original_data.columns = original_data.columns.str.lower()
+
+    # Rename variables to GETTSIM convention
+    var_names = {
+        "id": "p_id",
+        "yob": "geburtsjahr",
+        "mob": "geburtsmonat",
+        "regsray": "Rentenalter_Jahr",
+        "regsram": "Rentenalter_Monat",
+        "m_pflichtbeitrag": "m_pflichtbeitrag",
+    }
+    out = original_data[[*var_names]].rename(columns=var_names).copy()
+
+    # Create IDs
+    out["tu_id"] = out["p_id"]
+    out["hh_id"] = out["p_id"]
+
+    # Create variables needed in GETTSIM
+    out["weiblich"] = True
+    out["jahr"] = random.choice(YEARS)
```

Comment:
Why do we have a random choice here?

Reply:
My idea was: Since the original testcases do not have a year, I had to attach a year. However, if all tests have the same year attached to them, then we would only have one testcase. So to ensure that we test for different years I implemented this, but maybe there is a better solution?

Reply:
You can have the fixture taking arguments: https://docs.pytest.org/en/latest/example/parametrize.html#apply-indirect-on-particular-arguments

Reply:
Well should the tested outcome depend on the year? I am a bit surprised that it doesn't as the respective parameters probably change over time?

Reply:
They mostly (and in the test cases completely) depend on birth date not the year of the claim. The gettsim structure and the the tests currently always require a payout/taxation year , however.
It actually sound like a good solution to me to randomize -  maybe we could have a range of say 1990-2020 to make sure that we have the relevant tax and transfer structure established for the test cases

Reply:
We'll never want to have randomization in tests, would lead to failures that cannot be reproduced. Say you run it once and 1995 comes up (and fails), next time it does not and tests pass...

My suggestion above would just line up the evaluation year (used to calculate age, I guess) and the policy year, tests are still being run for the entire range of years in the file. Would there be a use case for anything else in the tests? *(as opposed to counterfactual experiments)*

Speaking of years, how come they go up to 2021? Because we are sure the policy environment did not change for anything that happened after Arthur's analysis/data prep stops? Must be way earlier given paper publication and data publication lags...

Reply:
> are still being run for the entire range of years in the file

thats good to know. So it's enough to use one example year.

> Speaking of years, how come they go up to 2021? Because we are sure the policy environment did not change for anything that happened after Arthur's analysis/data prep stops? Must be way earlier given paper publication and data publication lags..

The Seibold table contains the retirement ages up to cohort 1975. Laura is only using the columns of Regelaltersrente, abschlagsfreie Altersrente für langjährig Versicherte und Altersrente für Frauen. The information in these columns is up to date with all the policy changes currently known to me. 
 (not so the column for "besonders langjährig Versicherte"- thats why we do not use it) . For potential use of all columns we should stop evaluating in 2014.
I'd argue that policy uncertainty about the "Regelaltersgrenze" and "Altersgrenze für langjährig Versicherte" is mostly relevant for cohorts born after 1964 - as the adjustment rates are already set up to this cohort and the "Altersrente für Frauen" probably remains abolished.

Reply:
> They mostly (and in the test cases completely) depend on birth date not the year of the claim. 

Thanks for the explanation. I think, we can then just add one year (2010?) and run the tests for this year only.. probably best would be if we made a note/comment somewhere for which time frame we expect those policy rules to be in place.

> (not so the column for "besonders langjährig Versicherte"- thats why we do not use it)

Do you mean that we haven't implemented the full policy regime or that the test cases would be only valid until 2014? In the latter case, I think we should keep them and just set the evaluation year to some year before.

Reply:
I think there is value in keeping all years here because we might introduce seemingly innocuous changes that break these things at other points of the policy system.

Agreed that we should test the pre-2014 environment as extensively as possible.

Reply:
> I think there is value in keeping all years here because we might introduce seemingly innocuous changes that break these things at other points of the policy system.

Then, how about running all tests for a set of years (if that does not take too long to run)

Reply:
Fine, I guess that is how I meant "all years".

Just to be sure: Is there really nothing here that changed late in life beyond "Rente mit 63"? I.e., up to some year a birth cohort was expecting something that was changed later?

Reply:
well, that this something which is not considered in the functions yet

Reply:
all rules are established as they are known now

Reply:
they do work thereafter for the columns we considered. seibold's sample period is up to year 2014. therefore I guess we can be sure that all the retirement ages relevant up to this date are precise, for everything beyond that, they should be checked (especially for reforms that came up later). I checked for the three columns that we use now. 
We might be able to use more columns for all years. some of the remaining columns refer to pension types that -  to my knowledge - are not implemented yet.

Reply:
> We should open an issue

Will do

Reply:
Referencing #491 to be able to go back to this discussion.